operator/v1/types_network.go

```diff
@@ -35,6 +36,8 @@ type NetworkList struct {
 
 // NetworkSpec is the top-level network configuration object.
 type NetworkSpec struct {
+	OperatorSpec `json:",inline"`
```

Comment:
You should remove the existing `LogLevel` field now so there's no ambiguity between the `NetworkSpec` and `OperatorSpec` ones

Reply:
Oh, good catch.